proto_proxy.c

```diff
@@ -1910,8 +1910,8 @@ static int mcplib_hash_selector(lua_State *L) {
             lua_setfield(L, -2, "id");
             // TODO: need to get hostname (Separate from logical name?)
             // into the backend objects somehow.
-            lua_pushstring(L, "unknown");
-            lua_setfield(L, -2, "thing");
+            lua_pushstring(L, be->ip);
```

Comment:
you don't need the actual hostname?

Reply:
no, we use ip in place of hostname as well :)

Reply:
well that simplifies things in the meantime :) Should probably leave a comment saying that this is modified from the original algorithm and not actually using the hostname.